### PR TITLE
update readme

### PR DESCRIPTION
**Pull request by zhu733756**
- [ ] README.md updated with any new values or changes

### CODE REVIEW

File: elasticsearch/examples/kubesphere/Makefile [hunk 1]
```diff
@@ -3,12 +3,13 @@ default: test
 include ../../../helpers/examples.mk
 
 PREFIX := elasticsearch
+RELEASE := elasticsearch-data
```
Comment (benjaminhuo) [inline elasticsearch/examples/kubesphere/Makefile]: why adding `RELEASE ` ？It isn't used anywhere.

File: elasticsearch/examples/kubesphere/README.md [hunk 2]
```diff
@@ -137,17 +70,17 @@ uninstall:
...
 
 ```
 make install [-e NAMESPACE=xxx]
 ```
 
-#### patch
+#### Patch
 
-Skip this step if status of the pods is healthy
+Skip this step if all the states of the pods are normal.
```
Comment (benjaminhuo) [inline elasticsearch/examples/kubesphere/README.md]: Skip this step if all the states of the pods are normal.
=>
Skip this step if the states of all the pods are normal.

File: elasticsearch/examples/kubesphere/README.md [hunk 3]
```diff
@@ -137,17 +70,17 @@ uninstall:
...
 
-In my way, a pending status occured like below:
+In my case, pending status occured like below:
```
Comment (benjaminhuo) [inline elasticsearch/examples/kubesphere/README.md]: In my case, pending status occured like below:
=>
In my case, pending status looks like below:

File: elasticsearch/examples/kubesphere/README.md [hunk 4]
```diff
@@ -226,3 +159,84 @@ $ curl http://<your node ip>:30057/_cluster/health?pretty
...
+roles:
+  master: "false"
+  ingest: "true"
+  data: "true"
+  ml: "false"
+  remote_cluster_client: "false"
+
+
+# tolerations for data node
+tolerations:
```
Comment (benjaminhuo) [inline elasticsearch/examples/kubesphere/README.md]: No need to add toleration for master node.

ES cluster should not be scheduled to master node in the production scenario.

You can comment out the toleration part for debug purpose

File: elasticsearch/examples/kubesphere/README.md [hunk 5]
```diff
@@ -226,3 +159,84 @@ $ curl http://<your node ip>:30057/_cluster/health?pretty
...
+nodeGroup: "master"
+
+roles:
+  master: "true"
+  ingest: "false"
+  data: "false"
+  ml: "false"
+  remote_cluster_client: "false"
+
+# tolerations for master node
```
Comment (benjaminhuo) [inline elasticsearch/examples/kubesphere/README.md]: No need to add toleration for master node.

ES cluster should not be scheduled to master node in the production scenario.

You can comment out the toleration part for debug purpose

File: elasticsearch/examples/kubesphere/README.md [hunk 6]
```diff
@@ -226,3 +159,84 @@ $ curl http://<your node ip>:30057/_cluster/health?pretty
...
+
+# replicas 
+replicas: 3
+minimumMasterNodes: 2
+
+# Starting from version 7.11, Elasticsearch automatically sizes JVM heap based on a node's roles and total memory.
+# This configuration should not be modified.
+esJavaOpts: null
+
+# Adjust the limit of the Data node according to the data size, and the memory can be up to 32GI
```
Comment (benjaminhuo) [inline elasticsearch/examples/kubesphere/README.md:211]: Adjust the limit of the Data node according to the data size, and the memory can be up to 32GI
=>
Adjust the limit of the Data node according to the data size, the memory can be set up to 32Gi.

**Review by benjaminhuo — CHANGES_REQUESTED** [review]

File: elasticsearch/examples/kubesphere/Makefile (hunk 1 above)
Comment (zhu733756) [reply elasticsearch/examples/kubesphere/Makefile]: In actual, make test  need this params by `include ../../../helpers/examples.mk`. However, `make test` has some problem in test. Shell we remove this function?

Comment (benjaminhuo) [reply elasticsearch/examples/kubesphere/Makefile]: We can remove it for now, setting to `elasticsearch-data` while ignoring master node doesn't look good?

**Review by benjaminhuo — CHANGES_REQUESTED** [review]

File: elasticsearch/examples/kubesphere/Makefile (hunk 1 above)
Comment (zhu733756) [reply elasticsearch/examples/kubesphere/Makefile]: Ok. In this case, the `test` function take a http request to check whether the service of the elasticsearch-data nodes is ok or not.

Comment (benjaminhuo) [reply elasticsearch/examples/kubesphere/Makefile]: I see, you can comment this line out instead of removing it

Comment (zhu733756) [reply elasticsearch/examples/kubesphere/Makefile]: OK

**Review by benjaminhuo — APPROVED** [review]

**Pull request merged**